test/integration/scss/test/index.test.js

```diff
@@ -34,17 +34,18 @@ describe('SCSS Support', () => {
         env: { NODE_OPTIONS: shellQuote([`--require`, mockFile]) },
         stderr: true,
       })
+      let cleanScssErrMsg =
+        '\n\n' +
+        './styles/global.scss\n' +
+        "To use Next.js' built-in Sass support, you first need to install `sass`.\n" +
+        'Run `npm i sass` or `yarn add sass` inside your workspace.\n' +
+        '\n' +
+        'Learn more: https://err.sh/next.js/install-sass\n' +
+        '\n'
+
       expect(code).toBe(1)
       expect(stderr).toContain('Failed to compile.')
-      expect(stderr).toContain(
-        "To use Next.js' built-in Sass support, you first need to install `sass`."
-      )
-      expect(stderr).toContain(
-        'Run `npm i sass` or `yarn add sass` inside your workspace.'
-      )
-      expect(stderr).toContain(
-        'Learn more: https://nextjs.org/docs/messages/install-sass'
-      )
+      expect(stderr).toContain(cleanScssErrMsg)
```

Comment:
This should probably still use `toMatch` for an exact match, not `toContain`.

Or a better alternative is `toMatchInlineSnapshot` so you can drop the extra variable.  https://jestjs.io/docs/snapshot-testing#inline-snapshots

Reply:
I don't think that either`toMatchInlineSnapshot` or `toMatch` would be the right choice here, since the `stderr` string contains a lot of machine specific information such as absolute paths
i.e. 
`at new Input (/Users/baruchadi/projects/next/next.js/packages/next/node_modules/postcss/lib/input.js:24:13)`

and given that the string is about 60 lines long and has more than than 6500 characters (see example value here: https://gist.github.com/baruchadi/81be9d5b4d1f45d6f1f22b2195777a9a), I believe that `.toContain` would be our best bet at the moment. 
We can do `.toMatch` with some regex, but that would require us to make a lot of assumptions and the result would be very similar to `.toContain`.

**Note** that in the code, i'm checking for two new lines before and after our clean message. this should guarantee that the stack was removed from above and below our message.

see the diff between our error message and the original unedited one 
![image](https://user-images.githubusercontent.com/3421731/157109430-caa4eb03-08a3-4077-84e6-7ffb877cabde.png)

Reply:
Wasn't the point of this PR to only have https://gist.github.com/baruchadi/81be9d5b4d1f45d6f1f22b2195777a9a#file-stderr-output-L4-L8 ? That should always be the same.